### ISSUE TITLE
Stör ut larmet

### ISSUE THREAD

**Issue by mberndtsson** [Vaggan/com.edling.sectoralarm#4]
Hej

den senaste versionen stör ut larmet. Är problem att aktivera och avaktivera via fjärrkontrollerna. Kommer det snart en ny uppdatering?

**Comment by Vaggan** [comment]
Hej

Det har varit en del problem med appen men jag har inte hört något om att fjärrkontrollerna inte ska fungera (störs ut). Kan tyvärr inte testa detta själv eftersom jag inte har någon fjärrkontroll.

Jag har lagt ut en ny testversion nu ikväll, hoppas den fungerar bättre.
[https://homey.app/a/com.edling.sectoralarm/test/](https://homey.app/a/com.edling.sectoralarm/test/)

Återkom gärna om felen kvarstår så får jag kolla vidare på det.

**Comment by mberndtsson** [comment]
Hej

Det blir fortfarande problem om jag installerar appen i Home, det som händer är följande.
Push notiserna ifrån Sectors App slutar fungera, får ingen status på när man aktiverar/avaktiverar larmet.
Om man larmar av skalskyddet via fjärrkontrollerna så blinkar fortfarande manöver panelerna att skalskyddet är på.
Så det verkar som om appen i Homey på något sätt är i konflikt med Sectors egna app.

När man konfigurerar i Homey appen vilka uppdateringsfrekvens rekommenderar du? Har haft den på 30 sek.

**Comment by Vaggan** [comment]
Låter väldigt konstigt. 
Jag har ställt lägsta till 30 sekunder så den inte ska ”polla” data för ofta så ens konto blir låst. Optimalt så hade man velat ha den mycket lägre för att ”synka” med larmet så snabbt som möjligt. Men då låser Sector alarm ens konto. 
Kör du version 6 på homey och 0.1.4 på Sector appen? Testat att starta om appen och/eller homey?

**Comment by mberndtsson** [comment]
Yes kör senaste versionerna. Såg nu också att när man kollar i Sector appen så finns det inget under historik. Så fort man tar bort appen i Homey så fungerar allt igen. Väldigt skumt
